Library/Formula/libsamplerate.rb

```diff
@@ -26,6 +26,21 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make", "install"
+    if build.without? "libsndfile"
+      rm_f "#{bin}//sndfile-resample"
+    end
+  end
+
+  def caveats
+    s = ""
+    if build.without? "libsndfile"
+      s += <<-EOS.undent
+      The example program, "sndfile-resample", if not compiled with libsndfile
+      is completely broken and non-functional. (http://www.mega-nerd.com/libsndfile/).
+      Unless "libsndfile" is used as an option, "sndfile-resample" will be removed.
+      EOS
+      s
```

Comment:
The `s` here should be outside the `end` below. Check the `node` formula if you want an example :)